flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointFailureReason.java

```diff
@@ -44,6 +44,8 @@
 
 	CHECKPOINT_DECLINED_TASK_NOT_READY(false, "Checkpoint was declined (tasks not ready)"),
 
+	CHECKPOINT_DECLINED_TASK_CLOSING(false, "Checkpoint was declined (task's operators partially closed)"),
```

Comment:
I think this is missing a relevant entry in `CheckpointFailureManager` switch/case

Reply:
Ah jesus, wtf is this monstrosity. But fixed.